src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball ball = new Ball();
+        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
+            ball = new Lottery().getRandomBall();
```

Comment:
Do not create a Lottery inside the loop. You need 3 balls. Creating 3 Lotteries here is like visiting 3 different markets when you need to buy 3 milk bottles.